lib/chewy/railtie.rb

```diff
@@ -37,7 +37,8 @@ def migrate(*args)
       if app.sandbox?
         Chewy.strategy(:bypass)
       else
-        Chewy.strategy(:urgent)
+        strategy = Chewy.console_strategy || :urgent
+        Chewy.strategy(strategy)
```

Comment:
Let's initialize it properly in https://github.com/toptal/chewy/pull/780/files#diff-542f2b8cbb20f056db635819b1e65c367d4e8fdb9b7e6ae000ecda0c107f1127R52-R64